projects/step-frontend/src/lib/modules/timeseries/util/filter-utils.ts

```diff
@@ -26,7 +26,10 @@ export class FilterUtils {
     }
     let andFilters: (string | undefined)[] = items.map((item) => {
       let clause;
-      const attributeName = attributesPrefix ? `${attributesPrefix}.${item.attributeName}` : item.attributeName;
+      let attributeName = attributesPrefix ? `${attributesPrefix}.${item.attributeName}` : item.attributeName;
+      if (attributeName.includes('')) {
```

Comment:
Can you quickly tell me what this does? I would assume .includes('') is always true except if it is not a string (and then it will throw an error)

Reply:
@neogucky it had to be " " (white space). I fixed it